python/rmm/tests/test_rmm.py

```diff
@@ -942,3 +943,46 @@ def test_rmm_device_buffer_copy(cuda_ary, make_copy):
     result = db_copy.copy_to_host()
 
     np.testing.assert_equal(expected, result)
+
+
+@pytest.mark.parametrize("level", rmm.logging_level)
+def test_valid_logging_level(level):
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore", message="RMM will not log logging_level.TRACE."
+        )
+        warnings.filterwarnings(
+            "ignore", message="RMM will not log logging_level.DEBUG."
+        )
+        rmm.set_logging_level(level)
+        assert rmm.get_logging_level() == level
+        rmm.set_logging_level(rmm.logging_level.INFO)  # reset to default
+
+
+invalid_logging_levels = ["INFO", 3, "invalid", 100, None, 1.2345, [1, 2, 3]]
```

Comment:
suggestion (non-blocking): instead of parametrizing, you could define two pytest fixtures `invalid_level` and `level` (or `valid_level`).

Reply:
Hmm, fixtures don't seem to re-run the test for each item in a list it returns. I couldn't figure out how to get it working that way, and this way is really simple.

Reply:
For a fixture with multiple values you wouldn't return the list, you would parametrize the fixture itself. Although seemingly a bit more complex, in general there are many reasons to prefer fixtures over parametrize due to how pytest's model works. That said, most of them aren't super important for rmm given the size of the test suite, so no need to make this change.